dev/widgets.py

```diff
@@ -4,6 +4,39 @@
 import time
 from machine import Pin
 
+# TODO: Figure out best way to use constants for this
+# Alignment options:
+# CENTER = lv.ALIGN.CENTER
+# BOTTOM_L = lv.ALIGN.IN_BOTTOM_LEFT
+# BOTTOM_M = lv.ALIGN.IN_BOTTOM_MID
+# BOTTOM_R = lv.ALIGN.IN_BOTTOM_RIGHT
+# TOP_L = lv.ALIGN.IN_TOP_LEFT
+# TOP_M = lv.ALIGN.IN_TOP_MID
+# TOP_R = lv.ALIGN.IN_TOP_RIGHT
+# LEFT_M = lv.ALIGN.IN_LEFT_MID
+# RIGHT_M = lv.ALIGN.IN_RIGHT_MID
```

Comment:
Do we want to remove these commented out lines?

Reply:
No I want to keep these for future reference